### DISCUSSION TITLE
How do I remove an email from my account?

### DISCUSSION THREAD

**Discussion by Rod-in-NM** [community/community#43879] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My GitHub account has two email addresses associated with it. One is my personal email address and the other is my work email address. (I thought it best to have two, so I could recover if I encountered any problem.) Anyway, I've had this account for a few years and it's a free account.

Well, things at work are changing. We're supposed to migrate our repos to GitHub Enterprise sometime this year. Because of that I think it best to remove my work email address from this, my personal GitHub account. Whenever we get around to migrating our repos to GitHub, then I'll create a new GitHub account using my work email address.

What I'd like to know is how to go about doing this? Is it just the case of removing my work email address from my current GitHub account? Or is there anything else I must do?

**Comment by radar-blip** [comment, marked as answer]
I would think removing your work email would be enough. 
As long as you keep your personal email address as your primary email.

**Reply by Rod-in-NM** [reply]
Thank you; glad to know that nothing else is involved.

**Reply by radar-blip** [reply]
You're welcome. I re-read your question (just in case) and there's nothing else you should have to do. 
You should be all set!
